Casks/graalvm-ce-java16.rb

```diff
@@ -0,0 +1,48 @@
+cask 'graalvm-ce-java16' do
+  version '21.1.0'
+  sha256 'dafece9d03251304a6d9fc242862cfc08b85e2b8921d3b019a8a19b95af78e2c'
+
+  JVMS_DIR = '/Library/Java/JavaVirtualMachines'.freeze
+  TARGET_DIR = "#{JVMS_DIR}/#{cask}-#{version}".freeze
+
```

Comment:
This may be necessary (forgot why though, some bit in Homebrew):
```suggestion

# github.com/graalvm/graalvm-ce-builds was verified as official when first introduced to the cask
```